src/background/telemetry/app-insights-telemetry-client.ts

```diff
@@ -78,6 +78,7 @@ export class AppInsightsTelemetryClient implements TelemetryClient {
     }
 
     private initializeInternal(): void {
+        this.appInsights.context.operation.name = '';
```

Comment:
I'm guessing this affects both AI-Web and AI-Android right?
I'm curious... one of the links you posted mention that **"Operation Name... [is] usually the URL of the HTTP request."** (I'm paraphrasing here). Some time ago, we did have the url as part of our telemetry and we did work to remove it (for GDPR compliance). Now I'm a little worried about this finding. I don't recall we talk about this one while removing the url and I'm wondering if we do actually see any url from **operation.name** property on telemetry.

Reply:
@smoralesd based on what Fer showed me for web, it seems those URLs are relative to the chrome extension, e.g. something like 'background/background.html'. It's possible the electron URLs are this way because of how we're bundling canary; removing the field seems fair just in case